README.md

```diff
@@ -219,7 +221,75 @@ Flutter ile ilgili Türkçe makaleler kategorilerine göre ayrılmış halde bur
 - 🐦 [Flutter Türkiye](https://twitter.com/flutter_turkiye)
 - 🐦 [Flutter İzmir](https://twitter.com/FlutterIzmir)
 
+
+## İngilizce Kaynak Başlıkları
+- [Makaleler](#ingilizce-makaleler)
+- [Videolar](#ingilizce-videolar)
+
+## Ortak Kaynaklar
+- [Açık Kaynak Kodlar](#acık-kaynak-kodlar-eng)
+
+### <a name="ingilizce-makaleler"></a>İngilizce Makaleler ENG 📝
+Flutter ile ilgili İngilizce makaleler kategorilerine göre ayrılmış halde burada bulunabilir.
+
+#### Widgetlar
+
+- [Navigation Rail Widget](https://itnext.io/navigation-rail-widget-flutter-1-17-229f7c5d3215) - Muhammad Hamza
+- [Getting to the Bottom of Navigation in Flutter](https://medium.com/flutter/getting-to-the-bottom-of-navigation-in-flutter-b3e440b9386) - Hans Muller
+
+#### Kütüphaneler
```

Comment:
Bu kaynağa katkın için teşekkürler! Bunlar kütüphaneden ziyade, eğitici içerikler kategorisinde. Bunları ait oldukları kategorilere bölmek daha makul olanı gibi geliyor bana.